scripts/competitions/svcomp/esbmc-wrapper.py

```diff
@@ -256,7 +256,7 @@ def get_command_line(strat, prop, arch, benchmark, concurrency, dargs, esbmc_ci)
                  check_if_benchmark_contains_pthread(benchmark))
 
   if concurrency:
-    command_line += " --no-por --context-bound 2 "
+    command_line += " --smt-during-symex --smt-thread-guard --z3 --no-por --context-bound 2 "
```

Comment:
Whoa, wait.

Reply:
sorry, by mistake..I thought this was merged

Reply:
Done now.